### ISSUE TITLE
Compute separation: Unable to stop execution of cell calling locator() 

### ISSUE THREAD

**Issue by sanketd11** [att/rcloud#1685]
This issue is observed on RCloud-5 instance. Following are the steps to replicate the issue:
- call locator() in a cell below R cell that contains plots
- press stop button

The execution of locator() doesn't get terminated and shows one of the below outputs after clicking specified number of times 

![locator_rc5_stop_cell_status](https://cloud.githubusercontent.com/assets/9251570/10216881/40c5ed54-67fa-11e5-9c0c-9bde766d7d15.PNG)

![locator_rc5_stop_cell_status1](https://cloud.githubusercontent.com/assets/9251570/10216880/40c507e0-67fa-11e5-9140-51b8d68dd58e.PNG)

![locator_rc5_stop_cell_status2](https://cloud.githubusercontent.com/assets/9251570/10216882/40c69ede-67fa-11e5-8575-53a9f97291be.PNG)

**Comment by s-u** [comment]
R doesn't allow cancellation of locator by interrupt - that is true for regular R as well. The only way to cancel a locator is to press `<ESC>`. Since the UI knows about the modal state of a locator, it could map "Stop" to the locator cancellation.

**Comment by gordonwoodhull** [comment]
Yes - eventually. We should deal with this at the same time we deal with input cancellation #1196.
